internal/pkg/folder-package_test.go

```diff
@@ -42,3 +44,19 @@ func TestFolder_InstallTo(t *testing.T) {
 	_, err = os.Stat(filepath.Join(targetDir, "fake_test", "manifest.mf"))
 	assert.Nil(t, err)
 }
+
+func TestFolder_InstallToWithSetupError(t *testing.T) {
+	p, err := CreateFolderPackage("assets/fake-wrong-setup")
+	assert.NotNil(t, p)
+	assert.Nil(t, err)
+
+	targetDir := t.TempDir()
+	assert.Nil(t, err)
+
+	var previousValue = viper.GetBool(config.ENABLE_PACKAGE_SETUP_HOOK_KEY)
+	viper.Set(config.ENABLE_PACKAGE_SETUP_HOOK_KEY, true)
+	mf, err := p.InstallTo(targetDir)
+	assert.NotNil(t, err)
+	assert.Nil(t, mf)
+	viper.Set(config.ENABLE_PACKAGE_SETUP_HOOK_KEY, previousValue)
```

Comment:
This line won't be executed in case of an assert error.   

Let's wrap this in a function and use `defer` to call it.